staging/src/k8s.io/api/testdata/HEAD/apps.v1.DaemonSet.json

```diff
@@ -1616,6 +1616,10 @@
                   ]
                 }
               ]
+            },
+            "nodeInclusionPolicies": {
```

Comment:
Jordan recently authored a PR to minimize the changes in testdata. It's good to rebase the codebase if you haven't.

Reply:
Thanks Wei.